src/main/java/com/rakovets/course/java/core/practice/oop/principles/hero/Demon.java

```diff
@@ -0,0 +1,7 @@
+package com.rakovets.course.java.core.practice.oop.principles.hero;
+
+public class Demon  extends Enemy{
```

Comment:
"Violation: There isn't separating white spaces for curly brake
Solution: Add white spaces"